webpack/assets/javascripts/react_app/components/common/dates/RelativeDateTime.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedRelative, intlShape } from 'react-intl';
+
+const RelativeDateTime = (props, context) => {
+  const { date, defaultValue } = props;
+  if (date) {
+    const title = context.intl.formatDate(date, {
+      day: '2-digit',
+      month: 'short',
+      hour: '2-digit',
+      minute: '2-digit',
+      year: 'numeric',
+    });
+
+    /* eslint-disable react/style-prop-object */
+    return (
+      <span title={title}>
```

Comment:
this means it uses the jquery popover right? perhaps we should be explicit and use the pf one?

Reply:
No, this means it uses standard html attribute: https://www.w3schools.com/tags/att_title.asp
I don't think jQuery is involved here.

Reply:
I was referring to https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/foreman_tools.js#L61

Reply:
I see. I didn't see it would apply anywhere on the dates. It's probably because the components mounted after the tooltip initialization.

Reply:
right, but if you want to show the correct date (e.g. a minute after the page has been loaded) you would need a different way. I"m OK to leave this outside of this pr.